tests/integration/test_infrahub_client.py

```diff
@@ -293,3 +293,28 @@ async def test_create_branch(self, client: InfrahubClient, db: InfrahubDatabase,
     async def test_create_branch_async(self, client: InfrahubClient, db: InfrahubDatabase, init_db_base, base_dataset):
         task_id = await client.branch.create(branch_name="new-branch-2", wait_until_completion=False)
         assert isinstance(task_id, str)
+
+    # See issue #148.
```

Comment:
I asked myself this once: should we just put the entire link to the issue as a comment to a test that solves it? I think it makes sense to have it/

Reply:
It depends on our preferencies. Actually, I think I prefer full link too. It's a bit more verbose but it allows quicker access to corresponding issue, and it is even more relevant as we have now multiple repositories (at least sdk/server), and it should be easy to find/replace these links in case repositories names change.